corehq/apps/analytics/templates/analytics/initial/appcues.html

```diff
@@ -0,0 +1,22 @@
+{% load hq_shared_tags %}
+
+{% if ANALYTICS_CONFIG.DEBUG %}
+    {# Developer is working on analytics #}
+    {% initial_analytics_data 'appcues.apiId' ANALYTICS_IDS.APPCUES_ID %}
+    {% initial_analytics_data 'appcues.userId' request.couch_user.userID %}
+    {% initial_analytics_data 'appcues.email' request.couch_user.get_email %}
+    {% initial_analytics_data 'appcues.dateCreated' request.couch_user.created_on %}
```

Comment:
`created_on` is a human readable string (`2016-02-22T15:45:40.259164Z`) but Appcues wants a value in seconds since the epoch. `corehq/util/dates.py` has helpful functions.

Reply:
ooh, good catch